ports/mago/mago/patch/patchscript

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+# Set the log file
+LOGFILE="$PATCHDIR/patchlog.txt"
+
+# Redirect output and error to the log file
+exec > >(tee -a "$LOGFILE") 2>&1
+
+echo "PATCHDIR is set to: $PATCHDIR"
+
+# Exports
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/lib:$GAMEDIR/tools/libs:$LD_LIBRARY_PATH"
+export TOOLDIR="$PATCHDIR/tools"
+export TMPDIR="$PATCHDIR/tmp"
+export PATH=$PATH:$PATCHDIR/tools
+
+
+# Permissions
+chmod 666 /dev/uinput
+chmod 777 "$TOOLDIR/gmKtool.py"
+chmod 777 "$TOOLDIR/oggenc"
+
+cd "$PATCHDIR"
+
+process_game() {
+    # Move assets from gamedata folder
+    if [ -d "gamedata" ]; then
+        mv gamedata/* .
+        echo "Moved assets from gamedata to PATCHDIR."
+        rm -rf gamedata
+        echo "Removed gamedata directory."
+    else
+        echo "gamedata directory not found. Skipping asset move."
+    fi
+
+    # Define files to delete
+    files_to_delete="Steamworks_x64.dll steam_appid.txt steam_api64.dll MAGO.exe"
+
+    # Delete unnecessary files
+    for file in $files_to_delete; do
```

Comment:
Actually, `for` will loop on one entry which is file="Steamworks_x64.dll steam_appid.txt steam_api64.dll MAGO.exe". I suppose in the end it kind of work because you must have checked that the file are indeed deleted but it's odd :)